include/aspect/material_model/rheology/elasticity.h

```diff
@@ -160,6 +160,12 @@ namespace aspect
            * Double for fixed elastic time step value, read from parameter file
            */
           double fixed_elastic_time_step;
+
+          /**
+           * Double for amount of stabilization applied to elastic stresses
```

Comment:
I would rephrase this the following way: `A stabilization factor for the elastic stresses that influences how fast elastic stresses adjust to deformation. 1.0 is equivalent to no stabilization, and infinity is equivalent to not applying elastic stresses at all. The factor is multiplied with the computational time step to create a time scale.`

Writing useful documentation is a bit unintuitive, but it is essentially not at all about telling what this **is** (users can see that in the code), but what this is **for**.